### PR TITLE
block individual xblock

### PR DESCRIPTION
**Pull request by felipemontoya**
This PR makes it possible to put a seb restriction in an individual page.

It will correctly allow for blocking of a url of the form:
http://local.edly.io:8000/xblock/block-v1:Demo+Demo+test+type@vertical+block@d30d79a1f41445cdb6125de70a88ff7d?exam_access=&recheck_access=1&show_bookmark=0&show_title=0&view=student_view

Replaces #66

### CODE REVIEW

File: seb_openedx/templates/seb-xblock.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<script type="text/javascript">
+  (function() {
+    // If this view is rendered in an iframe within the learning microfrontend app
+    // it will report the height of its contents to the parent window when the
+    // document loads, window resizes, or DOM mutates.
+    if (window !== window.parent) {
+      document.body.className += ' view-in-mfe';
+      var lastHeight = window.offsetHeight;
+      var lastWidth = window.offsetWidth;
```
Comment (BryanttV) [inline seb_openedx/templates/seb-xblock.html]: Are these variables necessary?

File: seb_openedx/templates/seb-xblock.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      var contentElement = document.getElementById('content');
+
+      function dispatchResizeMessage(event) {
+        // Note: event is actually an Array of MutationRecord objects when fired from the MutationObserver
+        var isLoadEvent = event.type === 'load';
```
Comment (BryanttV) [inline seb_openedx/templates/seb-xblock.html]: Same as above

**Review by BryanttV — APPROVED** [review]

File: seb_openedx/templates/seb-xblock.html (hunk 1 above)
Comment (felipemontoya) [reply seb_openedx/templates/seb-xblock.html]: No, probably code that was used by the more complex function in the platform that I simplified to get this working

File: seb_openedx/templates/seb-xblock.html (hunk 2 above)
Comment (felipemontoya) [reply seb_openedx/templates/seb-xblock.html]: removed

**Pull request merged**